mcts/src/main/resources/MctsAuditModification.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE motech_data_services.nms_mcts_audit CHANGE importDate endImportDate DATE NULL;
```

Comment:
Instead of just importDate, from now on we will have startImportDate and endImportDate. So planned to modify the name. 
Will look for an alternative now